compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEConvertDeviceToControlPackets.cpp

```diff
@@ -0,0 +1,203 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <filesystem>
+
+#include "aie/AIEDialect.h"
+#include "iree-amd-aie/IR/AMDAIEDialect.h"
+#include "iree-amd-aie/IR/AMDAIEOps.h"
+#include "iree-amd-aie/Target/AMDAIERT.h"
+#include "iree-amd-aie/Transforms/Passes.h"
+#include "iree-amd-aie/aie_runtime/iree_aie_configure.h"
+#include "mlir/IR/AsmState.h"
+
+#define DEBUG_TYPE "iree-amdaie-convert-device-to-control-packets"
+
+using Path = std::filesystem::path;
+
+namespace mlir::iree_compiler::AMDAIE {
+
+namespace {
+LogicalResult convertDeviceToControlPacket(IRRewriter &rewriter,
+                                           xilinx::AIE::DeviceOp deviceOp,
+                                           const std::string &pathToElfs) {
+  AMDAIEDeviceModel deviceModel = getDeviceModel(deviceOp.getDevice());
+
+  // Start collecting transations.
+  TRY_XAIE_API_LOGICAL_RESULT(XAie_StartTransaction, &deviceModel.devInst,
+                              XAIE_TRANSACTION_DISABLE_AUTO_FLUSH);
+  if (failed(addAllAieElfs(deviceModel, deviceOp, Path{pathToElfs},
+                           /*aieSim=*/false)))
+    return failure();
```

Comment:
```suggestion
                           /*aieSim=*/false))) {
    return failure();
  }
```
Because the condition has multiple lines.